Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+pipeline {
+	agent any
+	options {
+		buildDiscarder(logRotator(numToKeepStr:'10'))
+	}
+	stages {
+		stage('Prepare') {
+			steps {
+				git url: 'https://github.com/eclipse/wildwebdeveloper.git'
+				cleanWs()
+				checkout scm
+			}
+		}
+		stage('Build') {
+			steps {
+				wrap([$class: 'Xvnc', useXauthority: true]) {
+					withEnv([]) { // Which environment variable we should use for node ?
+						withMaven(maven: 'apache-maven-latest', jdk: 'jdk1.8.0-latest', mavenLocalRepo: '.repository') {
+							sh 'mvn clean verify -Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true -PpackAndSign'
```

Comment:
If we don't have Xvnc, let's pass the `-DskipTests` flag to not run tests at the moment.